gazebo_plugins/include/gazebo_plugins/gazebo_ros_wheel_slip.hpp

```diff
@@ -24,9 +24,12 @@ namespace gazebo_plugins
 {
 class GazeboRosWheelSlipPrivate;
 
+/// Description:
+/// ------------
 /// A plugin for adjusting wheel slip parameters in gazebo.
 /// The plugin can set separate longitudinal and lateral wheel slip compliance
-/// parameters for separate wheel links.
+/// parameters for separate wheel links. THe following ROS parameters can be specified in
+/// the world file (These are optional, you may not may not declare them):
```

Comment:
style nit: prefer one line per sentence for documentation. This generally makes diffs more readable and in my experience does not have an affect on generated documentation.

I've also applied some wording suggestions:
```suggestion
/// parameters for separate wheel links.
/// The following ROS parameters can optionally be specified in the world file
```

I'm not sure what was meant by "you may not declare them". I guess you mean subclasses shouldn't _also_ try to declare these parameters, but that's a general rule for all ROS parameters so I'm not sure it's worth calling attention to it here.